src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -320,8 +320,8 @@ private void processFiles(List<File> files) throws CheckstyleException {
      * @throws CheckstyleException if error condition within Checkstyle occurs.
```

Comment:
`@return a sorted set of messages to be logged.` => `@return a sorted set of violations to be logged.`

Reply:
Done